api/v1alpha1/twemproxy_types.go

```diff
@@ -0,0 +1,108 @@
+package v1alpha1
+
+import (
+	"time"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/pointer"
+)
+
+var (
+	// Twemproxy defaults
+	// WARNING -> REVIEW THIS!!!
+	defaultTwemproxyLivenessProbe defaultProbeSpec = defaultProbeSpec{
+		InitialDelaySeconds: pointer.Int32Ptr(0),
+		TimeoutSeconds:      pointer.Int32Ptr(1),
+		PeriodSeconds:       pointer.Int32Ptr(5),
+		SuccessThreshold:    pointer.Int32Ptr(1),
+		FailureThreshold:    pointer.Int32Ptr(3),
+	}
+	// WARNING -> REVIEW THIS!!!
+	defaultTwemproxyReadinessProbe defaultProbeSpec = defaultProbeSpec{
+		InitialDelaySeconds: pointer.Int32Ptr(0),
+		TimeoutSeconds:      pointer.Int32Ptr(1),
+		PeriodSeconds:       pointer.Int32Ptr(5),
+		SuccessThreshold:    pointer.Int32Ptr(1),
+		FailureThreshold:    pointer.Int32Ptr(3),
+	}
+	// TODO: add requirements
```

Comment:
TODO?

Should be important adding resources to twemproxy sidecar, so HPA can work (right now HPA needs CPU set for all containers, otherwise can't calculate it)

Reply:
This is just a default value, you can specify resource requirements for Twemproxy in the Backend spec. We just don't have data right now to set defaults for this.

Reply:
OK, fair enough!